client/src/App.js

```diff
@@ -1,36 +1,52 @@
-import React, { useState, useEffect } from 'react';
-import axios from 'axios';
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Movie from "./Movies/Movie";
+import SavedList from "./Movies/SavedList";
+import { Route, Switch } from "react-router-dom";
+import MovieList from "./Movies/MovieList";
```

Comment:
Small detail: Try to group your imports together cohererently, for example all component imports as such
`import axios from 'axios';`
`import {Route, Switch} from 'react-router-dom'`
`import Movie from './Movies/Movie';`
`import SavedList from './Movies/SavedList';`
`import MovieList from './Movies/MoveList';`